.github/workflows/assign_new_issue_to_project

```diff
@@ -0,0 +1,16 @@
+name: Auto Assign Issues to Projects
+
+on: issues
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_to_project:
+    runs-on: ubuntu-latest
+    name: Assign topic to Project
+    steps:
+    - name: Assign issues with new-topic label to GitHub Project
+      uses: srggrs/assign-one-project-github-action@1.2.0
+      with:
+        project: 'https://github.com/orgs/Manta-Network/projects/8'
+        column_name: 'No Status'
```

Comment:
Probably better to just rename the `No Status` to `Backlog` instead of having them both.

Reply:
There's a difference.
No status means "needs decision on when"
Backlog means "declared not important for now"